int/api/network/network_manager.go

```diff
@@ -0,0 +1,51 @@
+package network
+
+import (
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/station/api/swagger/server/models"
+	"github.com/massalabs/station/api/swagger/server/restapi/operations"
+	"github.com/massalabs/station/pkg/config"
+)
+
+func NewSwitchNetworkHandler(networkManager *config.NetworkManager) operations.SwitchNetworkHandler {
+	return &switchNetworkHandler{networkManager: networkManager}
+}
+
+type switchNetworkHandler struct {
+	networkManager *config.NetworkManager
+}
+
+func (h *switchNetworkHandler) Handle(params operations.SwitchNetworkParams) middleware.Responder {
+	err := h.networkManager.SwitchNetwork(params.Network)
+	if err != nil {
+		return operations.NewSwitchNetworkNotFound().WithPayload(
+			&models.Error{
+				Code:    "404",
```

Comment:
it's supposed to be a constant here, like const errorNetworkNotFound = "Network-001", see how we did it for other errors